node/overseer/src/lib.rs

```diff
@@ -415,6 +415,7 @@ pub async fn forward_events<P: BlockchainEvents<Block>>(client: Arc<P>, mut hand
 	signal=OverseerSignal,
 	error=SubsystemError,
 	network=NetworkBridgeEvent<VersionedValidationProtocol>,
+	message_capacity=2048,
```

Comment:
Does this do the right thing? Which is increasing the bounded message channel for the dispute-coordinator only? And ideally only if there was no command line override?

Reply:
Yeah, this is not currently possible. I talked to @drahnr about this and there were some concerns about not having the same channel size for all subsystems. We can revisit this and implement the option of overriding it for specific subsystems.

Reply:
message_capacity=2048 is used if there is no cli flag present, but for _all_ subsystems. This is intentional, we don't want `n^2` number of tunable parameters :upside_down_face: